felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/Verge.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.kontrakter.felles.tilbakekreving
+
+import no.nav.familie.kontrakter.felles.PersonIdent
+import org.springframework.format.annotation.DateTimeFormat
+import java.time.LocalDateTime
+
+data class Verge(val vergeType: VergeType,
+                 @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+                 val gyldigFom: LocalDateTime,
+                 @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+                 val gyldigTom: LocalDateTime,
+                 val navn: String,
+                 val organisasjonNummer: String? = null,
+                 val personIdent: PersonIdent? = null) //kan aktoerId brukes istedenfor?
```

Comment:
kan vel kalle det fødselsnummer eller fnr?

Reply:
eller `ident`

Reply:
Jeg liker å bruke samme begrep for identifikator for en person, så jeg liker personIdent, så slipper vi varianter med fnr/fødselsnummer/ident/dnr/ rundtomkring.